libevmasm/Assembly.cpp

```diff
@@ -1676,6 +1699,8 @@ LinkerObject const& Assembly::assembleEOF() const
 				break;
 			case Tag:
 				ret.bytecode += assembleTag(item, ret.bytecode.size(), false);
+				// since no jumpdest is added, these tags do not contribute to the bytecode
+				addInstructionLocation.skip();
```

Comment:
This one is a bit annoying. At first I thought I'd just leave them in there and then we'd have an instruction with `start == end` in terms of bytecode sequence. But then again they also don't really have meaningful source location info.